libnmstate/netinfo.py

```diff
@@ -36,6 +36,9 @@ def show(include_status_data=False):
     if include_status_data:
         report['capabilities'] = capabilities()
 
+    report[Constants.ROUTES] = nm.ipv4.get_route_info()
```

Comment:
`report[Constants.ROUTES] = nm.ipv4.get_route_info() + nm.ipv6.get_route_info()` could work as well and be easier to read.

Reply:
Done.